flink-core/src/test/java/org/apache/flink/statistics/heavyhitters/LossyCountingTest.java

```diff
@@ -0,0 +1,129 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.statistics.heavyhitters;
+
+import org.junit.Test;
+
+import java.util.Map;
+import java.util.Random;
+
+import static org.junit.Assert.assertTrue;
+
+/*
+* Test the structure implemented for Lossy Counting
+*/
+
+public class LossyCountingTest {
+
+	static final double fraction = 0.05;
+	static final double error = 0.005;
+	static final int seed = 7362181;
+	static final Random r = new Random(seed);
+
+	@Test
+	public void testAccuracy() {
+
+		int numItems = 1000000;
+		long frequency = (int)Math.ceil(numItems* fraction);
+		long minFrequency = (int)Math.ceil(numItems* (fraction-error));
+
+		int[] xs = new int[numItems];
+		int maxScale = 20;
+
+		for (int i = 0; i < numItems; i++) {
+			double p = r.nextDouble();
+			if (p<0.2){
+				xs[i] = r.nextInt(5);
+			}else {
+				int scale = r.nextInt(maxScale);
+				xs[i] = r.nextInt(1 << scale);
+			}
+		}
+
+		LossyCounting lossyCounting = new LossyCounting(fraction,error);
+
+		for (int x : xs) {
+			lossyCounting.addObject(x);
+		}
+
+		long[] actualFreq = new long[1 << maxScale];
+		for (int x : xs) {
+			actualFreq[x]++;
+		}
+
+		System.out.println("Size of heavy hitters: "+lossyCounting.heavyHitters.size());
```

Comment:
can you use regular logging instead of system.outs for these messages?
We have different logging configurations for travis, local and IDE but we can not control system.outs with our logging settings.